std/assembly/util/string.ts

```diff
@@ -600,15 +597,9 @@ function fixmul(a: u64, b: u32): u64 {
 }
 
 // @ts-ignore: decorator
+@inline
 function pow10(n: i32): f64 {
-  // @ts-ignore: type
-  const hi = Powers10Hi.dataStart as usize;
-  // @ts-ignore: type
-  const lo = Powers10Lo.dataStart as usize;
-  return (
-    load<f64>(hi + ((n >> 5) << alignof<f64>())) *
-    load<f64>(lo + ((n & 31) << alignof<f64>()))
-  );
+  return load<f64>(powers10 + (n << alignof<f64>()));
```

Comment:
Can you add an `assert` here with a comment, just to be sure we'll never load OOB?

Reply:
Assed under `DEBUG` becouse `pow10` inlined in three places

Reply:
Also added tests

Reply:
Hmm, the `DEBUG` thing there's mostly for tricky RT stuff (and we might get rid of that one day), not so much for stdlib.

Reply:
In this case I prefer remove `assert` check at all. It seems it properly covered by tests. Wdyt?

Reply:
Hmm, yeah, I guess that's best. Can you add a comment there that the argument range is limited?

Reply:
Sure. done!